rkt/metadata_service.go

```diff
@@ -511,6 +533,7 @@ func handlePodSign(w http.ResponseWriter, r *http.Request) {
 func handlePodVerify(w http.ResponseWriter, r *http.Request) {
 	defer r.Body.Close()
 
+	w.Header().Add("Content-Type", "text/plain; charset=us-ascii")
```

Comment:
This (and other enconding comments) are updates due to changes in appc spec, see https://github.com/appc/spec/pull/451/files#diff-dfc352c454901ff45ae64e4331968944R317